src/go/k8s/pkg/resources/configmap.go

```diff
@@ -40,12 +40,14 @@ const (
 
 	cloudStorageCacheDirectory = "cloud_storage_cache_directory"
 
-	tlsDir   = "/etc/tls/certs"
-	tlsDirCA = "/etc/tls/certs/ca"
-
-	tlsAdminDir          = "/etc/tls/certs/admin"
-	tlsPandaproxyDir     = "/etc/tls/certs/pandaproxy"
-	tlsSchemaRegistryDir = "/etc/tls/certs/schema-registry"
+	tlsRedpandaDir         = "/etc/tls/certs"
+	tlsRedpandaDirCA       = "/etc/tls/certs/ca"
+	tlsAdminDir            = "/etc/tls/certs/admin"
+	tlsAdminDirCA          = "/etc/tls/certs/admin/ca"
+	tlsPandaproxyDir       = "/etc/tls/certs/pandaproxy"
+	tlsPandaproxyDirCA     = "/etc/tls/certs/pandaproxy/ca"
```

Comment:
could you add an explanation why we need to put ca in the extra folder? Basically explaining the fact that with injected issuer you have a different CA and it is different secret so have to be in different folder

Reply:
✅